ccloud-monitoring/docs/index.rst

```diff
@@ -9,6 +9,7 @@ Monitoring |ak-tm| Clients to Confluent Cloud
 
    Monitoring overview and setup <monitoring-overview>
    Producer scenarios <producer-scenarios>
+   Consumer scenarios <consumer-scenarios>
```

Comment:
Sorry for slipping this comment into this RP but WDYT about moving Additional resources to be after Clean up?

Reply:
Moved. Rerunning docs build rn to validate changes didn't break anything.